smithy-cli/build.gradle

```diff
@@ -213,8 +213,8 @@ tasks.register("_dumpArchive", Exec) {
 
 // Can't do CDS if the OS and architecture is not one of our targets.
 if (!imageOs.isEmpty()) {
-    tasks["_dumpArchive"].dependsOn("_createClassList")
-    tasks["runtime"].finalizedBy("_dumpArchive")
+    //tasks["_dumpArchive"].dependsOn("_createClassList")
+    //tasks["runtime"].finalizedBy("_dumpArchive")
```

Comment:
Whoops unrelated change. We do need to move creating the CDS archive to install-time rather than build time later though since the jsa files apparently uses absolute paths.